pkg/webui/console/views/device/index.js

```diff
@@ -99,7 +100,10 @@ const DeviceContainer = props => {
     [appId, devId, deviceSelector, linkSelector, mayViewLink],
   )
 
-  useEffect(() => () => dispatch(stopDeviceEventsStream(devId)), [dispatch, devId])
+  useEffect(
+    () => () => dispatch(stopDeviceEventsStream(combineDeviceIds(appId, devId))),
```

Comment:
@kschiffer what is the reason the function returning function here?

Reply:
When using `useEffect()`, the function returned by the effect function is executed when unmounting the component.